crates/nu-command/src/dataframe/lazy/mod.rs

```diff
@@ -1,7 +1,7 @@
 pub mod aggregate;
 mod collect;
 mod fetch;
-mod fill_na;
+mod fill_nan;
```

Comment:
This just replaces `fill_na.rs` with `fill_nan.rs`. But the file for `fill_na` remains? What is the intent here?

Should this be a breaking change that removes the `fill-na` command. Or should there be both? 
(If thats the case both need to be added to the default context in the `bind_command!()` invocation below)

Reply:
OK with the name change.
Please delete `fill_na.rs`